gtsfm/utils/pycolmap_utils.py

```diff
@@ -12,11 +12,19 @@ def get_pycolmap_camera(camera_intrinsics: Cal3Bundler) -> pycolmap.Camera:
 
     See https://colmap.github.io/cameras.html#camera-models for info about the COLMAP camera models.
     Both SIMPLE_PINHOLE and SIMPLE_RADIAL use 1 focal length.
+    
+    Note: the image width and image height values approximated below are dummy placeholder values.
+    For some datasets we have intrinsics (including principal point) where image height, image width
+    would not necessarily be 2 * cy, 2 * cx. However, image width and image height aren't used anywhere
```

Comment:
nit: might be a bit too verbose. you may also consider using "image dimensions" instead of "image height and image width"